src/scss/components/_button.scss

```diff
@@ -15,5 +15,21 @@
         border-inline-start-color: transparent;
       }
     }
+
+    // approximate browser default behaviour for focus outline
+    &:focus-visible {
+      --utrecht-button-border-radius: 2px;
+    }
+  }
+}
+
+@include mobile-only {
+  .openforms-theme {
+    .utrecht-button,
+    .utrecht-button-link {
+      display: flex;
+      inline-size: 100%;
```

Comment:
What is the reason to use inline-size instead of width?

Reply:
inline-size accounts for vertical writing directions like Japanese instead of assuming horizontal writing direction.

It's similar to the `padding-block-start` which maps to `padding-top` for horizontal writing and `padding-inline-end` which maps to `padding-left` right-to-left writing mode.